phpBB/phpbb/attachment/upload.php

```diff
@@ -296,8 +296,10 @@ protected function check_attach_quota()
 	 */
 	protected function check_disk_space()
 	{
-		if ($free_space = @disk_free_space($this->phpbb_root_path . $this->config['upload_path']))
+		if (function_exists('disk_free_space'))
 		{
+			$free_space = disk_free_space($this->phpbb_root_path);
```

Comment:
I would still keep the `@` in case we might run into some weird things like open_basedir restrictions
```suggestion
			$free_space = @disk_free_space($this->phpbb_root_path);
```

Reply:
No need to silent it again, we are first checking if the function exists.

Reply:
> I would still keep the `@` in case we might run into some weird things like open_basedir restrictions

Sorry... I missed your statement, should I check if open_basedir is in effect as well?

Reply:
Silencing it with `@` is not about whether this function exists, but whether to suppress (at least some) warning messages that might be shown by this. If open basedir restrictions are in effect, supressing this with `@` is valid and works also on PHP 8.x.